client/src/Components/Pages/SignUp/index.js

```diff
@@ -69,10 +142,14 @@ export default class SignUpForm extends Component {
         }).then(res => res.json())
           .then((res) => {
             if (res.status === true) {
-              alertify.dialog('alert').set({ transition: 'fade', message: 'You signed up in successfuly' }).setHeader('<h2>Success</h2>').show();
+              alertify.alert()
+                .setting({
+                  label: 'Ok',
+                  message: 'Hello ,Welcome to Unicornz.Our mission here is to help you achieve the creative confidence to be the very best you can, in whatever field you choose, and to take your place amongst the future creators, leaders and business founders of the world you want to see.Unicornz will connect you to people and companies who, combined with our programmes and work experience initiatives, will help you explore your skills and hobbies in the future of work.My advice to you, Azara? Take every opportunity you can. Even those you think may not be of interest. Note down your likes and your dislikes and gradually you will start creating your unique path to your unique working future.Enjoy your journey,Marie-Clare',
+                  onok() { window.location = '/Questionnaire'; },
```

Comment:
@donia98 This line looks like it is missing the `:`, and should be 
```onok: () => { window.location = '/Questionnaire'; },``` or something like that.  Maybe I'm missing something, because I thought this would've thrown an exception or not worked in some way. 🤷‍♂️

Reply:
Sounds good!